src/components/HomePage/ContactSection.js

```diff
@@ -0,0 +1,149 @@
+import React from "react";
+
+import {
+    LinkedIn,
+    Facebook,
+    GitHub,
+    Instagram,
+    Twitter,
+    Language,
+    Email,
+} from "@material-ui/icons";
+import { makeStyles, useTheme } from "@material-ui/core/styles";
+import Theme from "../../AppTheme";
+
+import {
+    Grid,
+    List,
+    ListItem,
+    Button,
+    Typography,
+} from "@material-ui/core";
+
+const useStyles = makeStyles((theme) => ({
+    parentContainer: {
+        backgroundColor: Theme.palette.dark.main,
+        color: Theme.palette.dark.contrastText,
+        padding: theme.spacing(4),
+    },
+    colorText: {
+        color: Theme.palette.tertiary.main,
+    },
+    backgroundColor: {
+        color: Theme.palette.dark.main,
+    },
+    primaryColor: {
+        color: Theme.palette.primary.main,
```

Comment:
You're receiving the theme as argument to be able to use it inside, don't import the theme manually